api/src/main/java/org/pmiops/workbench/access/AccessModuleServiceImpl.java

```diff
@@ -252,22 +251,15 @@ public static Timestamp deriveExpirationTimestamp(Timestamp completionTime, Long
   private boolean isModuleEnabledInEnvironment(AccessModule module) {
     final AccessConfig accessConfig = configProvider.get().access;
 
-    switch (module) {
-      case ERA_COMMONS:
-        return accessConfig.enableEraCommons;
+    return switch (module) {
+      case ERA_COMMONS -> accessConfig.enableEraCommons;
         // RT Compliance training  and CT Compliance training modules are
         // controlled by the same feature flag COMPLIANCE_TRAINING
-      case COMPLIANCE_TRAINING:
-      case CT_COMPLIANCE_TRAINING:
-        return accessConfig.enableComplianceTraining;
-      case IDENTITY:
-        return accessConfig.enableRasLoginGovLinking;
-      case RAS_LINK_LOGIN_GOV:
-        return accessConfig.enableRasLoginGovLinking;
-      case RAS_LINK_ID_ME:
-        return accessConfig.enableRasIdMeLinking;
-      default:
-        return true;
-    }
+      case COMPLIANCE_TRAINING, CT_COMPLIANCE_TRAINING -> accessConfig.enableComplianceTraining;
+      case RAS_LINK_LOGIN_GOV -> accessConfig.enableRasLoginGovLinking;
+      case RAS_LINK_ID_ME -> accessConfig.enableRasIdMeLinking;
+      case IDENTITY -> accessConfig.enableRasLoginGovLinking || accessConfig.enableRasIdMeLinking;
```

Comment:
This code change should not cause any functional changes given that these 2 values happen to always be equal.  However: this better expresses when we want Identity to be required.